index.js

```diff
@@ -0,0 +1,376 @@
+	multipleStatements: true
+});
+// function parseCookies(req) {
+// 	var list = {},
+// 		rc = req.headers.cookie;
+
+// 	rc &&
+// 		rc.split(";").forEach(cookie => {
+// 			var parts = cookie.split("=");
+// 			list[parts.shift().trim()] = decodeURI(parts.join("="));
+// 		});
+
+// 	return list;
+// }
+// var hall_id = parseCookies(req).hall_id;
+app.get("/", (req, res) => {
+	if (req.session.user) {
+		res.redirect("/dashboard");
+	} else res.render("index");
+});
+app.get("/login", (req, res) => {
+	if (req.session.user) {
+		res.redirect("/dashboard");
+	} else res.render("login");
+});
+app.get("/dashboard", (req, res) => {
+	if (req.session.user) {
+		if (req.session.user.role === "teacher") {
+			con.query(
+				"select * from booking where user_id= ?",
+				req.session.user.id,
+				function(err, booking_data) {
+					con.query(
+						"select * from events where user_id= ?",
+						req.session.user.id,
+						function(err, events_data) {
+							res.render("dashboard_teacher", {
+								res: req.session.user,
+								booking_data,
+								events_data
+							});
+						}
+					);
+				}
+			);
+		} else if (req.session.user.role === "dean") {
+			con.query("select * from booking;", function(err, booking_data) {
+				con.query("select * from events;", function(err, events_data) {
+					res.render("dashboard_dean", {
+						res: req.session.user,
+						booking_data,
+						events_data
+					});
+				});
+			});
+		} else if (req.session.user.role === "facility") {
+			res.render("dashboard_facility");
+		} else {
+			res.redirect("/login");
+		}
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/dashboard_facility_card", (req, res) => {
+	if (req.session.user) {
+		con.query("select * from booking;", function(err, booking_data) {
+			con.query("select * from events;", function(err, events_data) {
+				res.render("dashboard_facility_card", {
+					res: req.session.user,
+					booking_data,
+					events_data
+				});
+			});
+		});
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/dashboard_facility_table", (req, res) => {
+	if (req.session.user) {
+		con.query("select * from booking;", function(err, booking_data) {
+			con.query("select * from events;", function(err, events_data) {
+				res.render("dashboard_facility_table", {
+					res: req.session.user,
+					booking_data,
+					events_data
+				});
+			});
+		});
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/request", (req, res) => {
+	if (req.session.user) {
+		res.render("request");
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/requestUpdate", (req, res) => {
+	if (req.session.user) {
+		con.query("select * from booking where id=4;", function(
+			err,
+			booking_data
+		) {
+			con.query("select * from events where id=30;", function(
+				err,
+				events_data
+			) {
+				console.log({ booking_data }, { events_data });
+				res.render("requestUpdate", {
+					res: req.session.user,
+					booking_data,
+					events_data
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'events_data' is not in camel case. (camelcase)](https://app.codacy.com/app/AravindVNair99/Hall-Management-System/pullRequest?prid=4592231)